plugins/test-coverage/skills/test-structure/unified_analyzer.py

```
@@ -0,0 +1,653 @@
+        # Discover files - focus on e2e tests only
+        test_files_paths, source_files_paths = discover_files(self.source_dir, self.language, e2e_only=True)
+
+        # Parse files
+        test_files = []
+        source_files = []
+
+        for path in test_files_paths:
+            try:
+                test_file = parse_test_file(path, self.language)
+                test_files.append(test_file)
+            except Exception as e:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)